index.js

```diff
@@ -231,4 +257,30 @@ class NatureRemoAircon {
     return [aircon];
   }
 
+  getTargetTemperatureStep() {
+    const modes = this.record.aircon.range.modes;
+    const temperatures = Object.values(modes)[0].temp.map(string => parseInt(string)).sort();
+    return temperatures[1] - temperatures[0];
+  }
+
+  getMinTargetTemperature() {
+    return Math.min(...this.getAllTemperatures());
+  }
+
+  getMaxTargetTemperature() {
+    return Math.max(...this.getAllTemperatures());
+  }
+
+  getAllTemperatures() {
+    let allTemperatures = [];
+    const modes = this.record.aircon.range.modes;
+
+    for (const mode of Object.values(modes)) {
```

Comment:
* Please fix so that temperatures are only gathered from `temp` under `cool` / `warm` mode.  Also, ignore empty string in `temp`.
* `Object.values` requires Node v7. As Homebridge itself supports Node v4.3.2, could you fix the code to work with older versions of Node, e.g. by using `require('object.values')`? (Personally I'm running Homebridge with Node v6.14.3, which is a common version used in CentOS/RHEL 7.)